src/main/java/org/elasticsearch/index/query/IdsQueryBuilder.java

```diff
@@ -19,51 +19,45 @@
 
 package org.elasticsearch.index.query;
 
-import com.google.common.collect.Iterables;
-
+import com.google.common.collect.Sets;
 import org.apache.lucene.queries.TermsQuery;
 import org.apache.lucene.search.Query;
+import org.elasticsearch.cluster.metadata.MetaData;
+import org.elasticsearch.common.Nullable;
 import org.elasticsearch.common.io.stream.StreamInput;
 import org.elasticsearch.common.io.stream.StreamOutput;
-import org.elasticsearch.common.io.stream.Streamable;
 import org.elasticsearch.common.lucene.search.Queries;
 import org.elasticsearch.common.xcontent.XContentBuilder;
 import org.elasticsearch.index.mapper.Uid;
 import org.elasticsearch.index.mapper.internal.UidFieldMapper;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Out of curiosity, and to avoid future collisions, is this intentional? My IDE (I know, I know...) changes "*" imports back to concrete classes when re-organizing imports.